src/run.cu

```diff
@@ -1,6 +1,10 @@
 #include <dlfcn.h>
 #include <errno.h>
 #include <stdio.h>
+#include <dirent.h>
+#include <sys/stat.h>
+#include <string.h>
```

Comment:
Is `string.h` still being used? I don't see it in the changes

Reply:
I know `strlen` is being called somewhere but jump to definition is not working for me in this file